debian/postinst

```diff
@@ -141,6 +153,11 @@ case "$1" in
           fi
       fi
 
+      # UA service PPAs support all ubuntu releases, no need to
+      # specialize apt config filenames per ubuntu release.
+      redact_ubuntu_release_from_ua_apt_filenames $APT_SRC_DIR
+      redact_ubuntu_release_from_ua_apt_filenames $APT_PREFERENCES_DIR
```

Comment:
This didn't seem to rename the expected /etc/apt/preferences.d/ubuntu-esm-infra-trusty -> /etc/apt/preferences.d/ubuntu-esm-infra. We have a bug to sort here as that "trusty" preference file is still left intact after a ua attach <token>